docs/case-worker-guide.md

```diff
@@ -40,3 +40,35 @@ After the code is successfully issued, the `short code` will be displayed to be
 The unique identifier can be used later to confirm if the verification code was used or not.
 
 ![issue code](images/users/issue02.png "view code")
+
+## Bulk issue verification codes
+
+If [enabled in the realm](/realm-admin-guide.md#bulk-issue-codes), there will be a menu option to bulk issue codes.
+
+![bulk issue menu](images/users/menu_bulk_issue.png "bulk issue menu")
+
+This allows the user to upload a .csv file and issue many codes at once to a list of patient phone numbers and their associated test date. The bulk uploader assumes that all results are `positive`.
+
+The bulk uploader is written as a javascript client. Rather than uploading the file, it parses on the client and makes requests to the issue API. This allows the server to discard phone numbers after sending the SMS without storing them. The uploader may be canceled and resumed, respects the server's rate-limiting throttle, and prevents the user from sending an SMS to the same phone twice using a retry code.
+
+### CSV Format
+`patient phone`,`test date`, [optional] `symptom date`
+
+* The patient phone should be in [E.164 format](https://www.twilio.com/docs/glossary/what-e164).
```

Comment:
Replace should with must / next line too

Reply:
done